tests/checks/performance/class_performance_imbalance_test.py

```diff
@@ -0,0 +1,197 @@
+import typing as t
+import re
+from sklearn.ensemble import AdaBoostClassifier
+from hamcrest.core.matcher import Matcher
+from hamcrest import (
+    assert_that, instance_of, has_property, all_of,
+    equal_to, has_entries, only_contains, calling, raises
+)
+
+from deepchecks import Dataset, CheckResult, ConditionCategory
+from deepchecks.checks import ClassPerformanceImbalanceCheck
+from deepchecks.utils import DeepchecksValueError
+
+from tests.checks.utils import equal_condition_result
+
+
+def test_class_performance_imbalance(
```

Comment:
Add annotations for each test like in other tests:
```
# Arrange
# Act
# Assert
```
right now I can't quickly understand what is being tested in each test

Reply:
updated